python/ray/serve/scripts.py

```diff
@@ -800,15 +797,12 @@ def build_app_config(import_path: str, name: str = None):
             )
 
         deployments = pipeline_build(app, name)
-        ingress = get_and_validate_ingress_deployment(deployments)
         schema = ServeApplicationSchema(
             name=name,
-            route_prefix=ingress.route_prefix,
+            route_prefix="/",
```

Comment:
```suggestion
            route_prefix=f"/{name}",
```

I think this will need to be a distinct generated route prefix for each application because the pydantic model will fail to parse if every route prefix is "/".

Reply:
oh you're right. don't we default it to `/{name}` somewhere else already?

Reply:
Ok I changed it to do `/{name}` if there are multiple apps, else just `/`

Reply:
nice lgtm